store/store_test.go

```diff
@@ -31,6 +31,8 @@ func TestStore(t *testing.T) {
 
 	assert.Equal(t, *store.tailHeader.Load(), suite.Head())
 
+	assert.Equal(t, *store.tailHeader.Load(), suite.Head())
```

Comment:
Is this and the two above asserts are the same?

Reply:
Hm, this happened after the rebase. Curious how it slipped here again.

Reply:
There is one more above lol